build.gradle

```diff
@@ -319,45 +324,27 @@ subprojects { subproj ->
         finalizedBy jacocoTestReportTask
     }
 
-    // TODO: These should be added to the CDK or to the individual projects that need them:
```

Comment:
Done. The dependencies have been hardcoded in the CDK, which no longer relies on deps.toml. We want to avoid significant version changes sneaking into the CDK.